### DISCUSSION TITLE
Scorecard Announcements September 21, 2022

### DISCUSSION THREAD

**Discussion by randytpierce** [NOAA-GSL/MATS#772] [Announcements]
## 👋 Welcome!
  We're using GitHub's Discussions as the central location for communication related to the Interactive Scorecard application. We're hoping that the majority of project-related communications can happen here so that, if needed, we can tag other people in for their input. 

We've created the following categories:
  * **Announcements**: Category for important communications from the project to others in the community.
  * **Development**. Category for regular conversations within the team about the development of the application.
  * **Ideas**. Category for suggestions and requests from stakeholders.
  * **Help**. Category for users to ask questions and get help with the software.
  * **Scorecard**. Category for specific questions and comments about the look and feel or operation of the prototype scorecard

**Comment by randytpierce** [comment]
Here is the github [documentation](https://docs.github.com/en/discussions)  for using github discussions. We are on this journey together (it is new to most of us) so you may want to glance at the guides.

One hint, if you would like to follow along with any discussion you can "subscribe" to the discussion in the watch pull down at the top right of the repository page. See the image below. You can choose the level of activity for which you would like to be notified.

 ![Screen Shot 2022-09-19 at 10 53 04 AM](https://user-images.githubusercontent.com/25647166/191071967-d18b6ad5-1937-49cf-99ce-69ea94f7691f.png)

**Comment by bonnystrong** [comment]


**Reply by randytpierce** [reply]
Thank you, yes. I blatantly stole this from Evan and Ian.

**Comment by randytpierce** [comment]
Following is a screenshot of the scorecard app. We are working on a continuous deployment mechanism and a prototype server for hosting the scorecard, but that isn't finished yet. For now we will have to demonstrate on a development machine and use screenshots to capture progress.

![Screen Shot 2022-09-20 at 5 01 06 PM](https://user-images.githubusercontent.com/25647166/191380056-f29e7f47-91cd-4026-a2c1-f77acb0543c0.png)

**Reply by randytpierce** [reply]
Where we are at this time : the scorecard app is now basically a new mats app, except that it gets its metadata from an API instead of reading the individual databases of each underlying app. The api is implemented in the MATS common code so every MATS app inherits it. The front end of the scorecard app is functional now and behaves much the same as a standard MATS app. As soon as we have a VM to deploy it on we will deploy it and make it available for scrutiny. Some notes about the scorecard app....

1. The places that MATS apps would say "Curve" are "Row" in a scorecard.
2. The "dates" and the "repeat" selectors are placeholders for the new selectors which aren't implemented yet.
3. We haven't yet decide how to save and restore scorecards. Those buttons are missing
4. We need to implement a mechanism for someone to "own" a scorecard. This will require implementing authorization of some sort.
